src/metatensor/models/experimental/soap_bpnn/model.py

```diff
@@ -22,13 +22,7 @@ class MLPMap(torch.nn.Module):
     def __init__(self, all_species: List[int], hypers: dict) -> None:
         super().__init__()
 
-        activation_function_name = hypers["activation_function"]
-        if activation_function_name == "SiLU":
-            self.activation_function = torch.nn.SiLU()
-        else:
-            raise ValueError(
-                f"Unsupported activation function: {activation_function_name}"
-            )
+        self.activation_function = torch.nn.SiLU()
```

Comment:
we want to remove the possibility of changing activation functions?
For me it is fine I just ask

Reply:
Yes for now. I think `torch.nn.SiLU` is enough.